### ISSUE TITLE
0.26.0 WebSocket does not fire onopen 

### ISSUE THREAD

**Issue by egorovsa** [nwjs/nw.js#6231]
v0.26.0 WebSocket does not work with numbers of connections more than 2.
v0.25.4 is everything ok

for instance we use simple test code

```
                let socket1 =  new WebSocket('wss://echo.websocket.org');
		let socket2 =  new WebSocket('wss://echo.websocket.org');
		let socket3 =  new WebSocket('wss://echo.websocket.org');
		let socket4 =  new WebSocket('wss://echo.websocket.org');

		socket1.onopen = ()=>{
			console.log('Socket 1 open');
		}

		socket1.onclose = () =>{
			console.log('Socket 1 close');
		}	

		socket2.onopen = ()=>{
			console.log('Socket 2 open');
		}

		socket2.onclose = () =>{
			console.log('Socket 2 close');
		}	

		socket3.onopen = ()=>{
			console.log('Socket 3 open');
		}

		socket3.onclose = () =>{
			console.log('Socket 3 close');
		}	

		socket4.onopen = ()=>{
			console.log('Socket 4 open');
		}

		socket4.onclose = () =>{
			console.log('Socket 4 close');
		}	
```

0.25.4 console.log

```
index.html:16 Socket 1 open
index.html:24 Socket 2 open
index.html:32 Socket 3 open
index.html:40 Socket 4 open
```

0.26.0 console.log only one session
```
index.html:16 Socket 1 open
```

http://prntscr.com/h1snjb 0.25.4
http://prntscr.com/h1smz0 0.26.0

**Comment by Christywl** [comment]
I can reproduce this issue on Linux with nwjs-sdk-v0.26.0, but there are two sessions in my device.
```
Socket 1 open
Socket 1 open
```

**Comment by rogerwang** [comment]
Thanks. Will check this after 0.26.1 release, which is scheduled today.

**Comment by rogerwang** [comment]
This is because upstream ships WebSocket throttling in 62. 
https://docs.google.com/document/d/1iR3XMIQukqlXb6ajIHE91apHZAxyF_wvRoB5JGeJYPs/edit#
Will disable this in NW.

**Comment by rogerwang** [comment]
This is fixed in git and will be available in the next nightly build.

**Comment by rogerwang** [comment]
The fix was released in 0.26.1

**Comment by egorovsa** [comment]
Thank you very much for your help. Now it works great!
